src/item.cpp

```diff
@@ -1397,6 +1397,10 @@ std::string item::info( bool showtext, std::vector<iteminfo> &info ) const
             info.push_back( iteminfo( "DESCRIPTION",
                                       _( "* This piece of clothing is designed to keep you <info>dry</info> in the rain." ) ) );
         }
+        if( is_armor() && has_flag( "RESTRICT_HANDS" ) ) {
+            info.push_back( iteminfo( "DESCRIPTION",
+                                      _( "* This piece of clothing hinders using both hands when wielding weapons." ) ) );
```

Comment:
"hinders using both hands" sounds like it hinders using both the left and the right hand.
I'd try something like "hinders usage of two-handed weapons".